app/src/lib/ui/Manager/ImportBeneficiaries.svelte

```diff
@@ -222,92 +190,37 @@
 		return pros;
 	}
 
-	function getStatusForStructureAssignement(structureId: string, pros: Professional[]): string {
-		if (pros.map((pro) => pro.structureId).includes(structureId)) {
-			return 'done';
-		}
-		return 'pending';
-	}
-
-	const client = getClient();
-
 	async function handleSubmit() {
 		insertInProgress = true;
 		insertResult = [];
+		const response = await importBeneficiariesApi(beneficiariesToImport);
 
-		const labels = beneficiariesToImport.flatMap(
-			(beneficiary) => beneficiary.wantedJobs?.split(',').map((s) => s.trim()) ?? []
-		);
-		const romeCodesResult = await client
-			.query<GetRomeCodesQuery>(GetRomeCodesDocument, { search: '', labels })
-			.toPromise();
-
-		const labelToRomeCode = romeCodesResult.data?.batch.reduce(
-			(acc, { id, label }) => ({ ...acc, [label]: id }),
-			{}
-		);
-		const romeCodeMatcher = stringToRomeCode(labelToRomeCode);
-		for (const beneficiary of beneficiariesToImport) {
-			const { uid, valid, proEmails = '', structureNames = '', ...benef } = beneficiary;
-			const accountIds = proEmailsToPros(proEmails).map(({ account }) => account.id);
-			const members = accountIds.map((accountId) => ({ memberType: 'referent', accountId }));
-			const structureIds = structureNamesToStructure(structureNames).map(({ id }) => id);
-			const structs = structureIds.map((structureId) => ({
-				structureId,
-				status: getStatusForStructureAssignement(structureId, proEmailsToPros(proEmails)),
-			}));
-			const payload = {
-				...benef,
-				workSituation: stringToWorkSituation(benef.workSituation),
-				rightRsa: stringToRightRsa(benef.rightRsa),
-				rightAre: stringToBool(benef.rightAre),
-				rightAss: stringToBool(benef.rightAss),
-				rightBonus: stringToBool(benef.rightBonus),
-				rightRqth: stringToBool(benef.rightRqth),
-				geographicalArea: stringToGeographicalArea(benef.geographicalArea),
-				educationLevel: stringToEducationLevel(benef.educationLevel),
-				members,
-				structures: structs,
-				wantedJobs:
-					beneficiary.wantedJobs
-						?.split(',')
-						.map((s) => s.trim())
-						.map(romeCodeMatcher)
-						.filter(Boolean)
-						.map((rome_code_id) => ({
-							rome_code_id,
-						})) || [],
-			};
-			await Promise.all([
-				inserter(payload),
-				new Promise((resolve) => {
-					setTimeout(resolve, 500);
-				}),
-			]);
-			let errorMessage = "Une erreur s'est produite, le bénéficiaire n'a pas été importé.";
-			if (
-				/uniqueness.*firstname_lastname_date_of_birth_unique_idx/i.test(insertStore.error?.message)
-			) {
-				errorMessage =
-					'Un bénéficiaire avec les mêmes prénom, nom et date de naissance existe déjà sur ce territoire.';
-			} else if (/uniqueness.*beneficiary_internal_id_key/i.test(insertStore.error?.message)) {
-				errorMessage = "Un bénéficiaire avec le même 'identifiant SI' existe déjà.";
-			}
+		const insertOperation = await response.json();
```

Comment:
je propose de renommer 
```suggestion
		const insertOperationResponse = await response.json();
```